### PR TITLE
cached-nix-shell: 0.1.3 -> 0.1.4

### PR DESCRIPTION
**Pull request by xzfc**
###### Motivation for this change

New release.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by nh2 — APPROVED** [review]

File: pkgs/tools/nix/cached-nix-shell/default.nix [hunk 1]
```diff
@@ -28,16 +28,9 @@ in rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ ronn ];
 
-  postBuild = ''
-    ronn -r cached-nix-shell.1.md
-  '';
+  postBuild = "make -f nix/Makefile post-build";
```
Comment (SuperSandro2000) [inline pkgs/tools/nix/cached-nix-shell/default.nix:31]:
```suggestion
  postBuild = ''
    make -f nix/Makefile post-build
  '';
```

File: pkgs/tools/nix/cached-nix-shell/default.nix [hunk 2]
```diff
@@ -28,16 +28,9 @@ in rustPlatform.buildRustPackage rec {
...
 
-  postInstall = ''
-    mkdir -p $out/lib $out/share/cached-nix-shell $out/share/man/man1 $out/var/empty
-    cp $releaseDir/build/cached-nix-shell-*/out/trace-nix.so $out/lib
-    cp rcfile.sh $out/share/cached-nix-shell/rcfile.sh
-    cp cached-nix-shell.1 $out/share/man/man1
-  '';
+  postInstall = "make -f nix/Makefile post-install";
```
Comment (SuperSandro2000) [inline pkgs/tools/nix/cached-nix-shell/default.nix:33]:
```suggestion
  postInstall = ''
    make -f nix/Makefile post-install
  '';
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 94567` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cached-nix-shell</li>
  </ul>
</details>

**Review by SuperSandro2000 — APPROVED** [review]
Output looks good:

```
tree results/cached-nix-shell/
 cached-nix-shell
├──  bin
│  └──  cached-nix-shell
├──  lib
│  └──  trace-nix.so
├──  libexec
│  └──  cached-nix-shell
│     └──  nix-shell ⇒ ../../bin/cached-nix-shell
├──  share
│  ├──  cached-nix-shell
│  │  └──  rcfile.sh
│  └──  man
│     └──  man1
│        └──  cached-nix-shell.1.gz
└──  var
   └──  empty
```

**Pull request merged**